src/Sulu/Bundle/MediaBundle/Media/Manager/DefaultMediaManager.php

```diff
@@ -150,7 +150,7 @@ public function __construct(
         MediaRepositoryInterface $mediaRepository,
         CollectionRepositoryInterface $collectionRepository,
         UserRepositoryInterface $userRepository,
-        ObjectManager $em,
+        EntityManager $em,
```

Comment:
i have use the `getReference` function of the entity manager to set the new collection of a media (see here: https://github.com/sulu-cmf/sulu/pull/805/files#diff-74482161f7eee6374460392c3c4c4441R798)